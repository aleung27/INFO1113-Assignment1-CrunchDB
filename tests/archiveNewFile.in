SET ...,./,/ 8
SET ? 2
SNAPSHOT
SNAPSHOT
SNAPSHOT
SNAPSHOT
ARCHIVE 3 archiveNewFile.txt
BYE
