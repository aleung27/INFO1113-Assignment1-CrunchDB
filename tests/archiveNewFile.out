> ok

> ok

> saved as snapshot 1

> saved as snapshot 2

> saved as snapshot 3

> saved as snapshot 4

> ok

> bye
